perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1138,6 +1139,21 @@
 	 */
 	RichMember createSponsoredMember(PerunSession session, Vo vo, String namespace, Map<String, String> name, String password, User sponsor) throws PrivilegeException, AlreadyMemberException, LoginNotExistsException, PasswordCreationFailedException, ExtendMembershipException, WrongAttributeValueException, ExtSourceNotExistsException, WrongReferenceAttributeValueException, UserNotInRoleException, PasswordStrengthException, InvalidLoginException;
 
+	/**
+	 * Creates a new sponsored Members (with random generated passwords) and its User.
```

Comment:
This message is little confusing. You can probably skip the whole part with "and its User".

Reply:
Fixed.